packages/actor-extract-links-solid-type-index/lib/ActorExtractLinksSolidTypeIndex.ts

```diff
@@ -195,9 +250,14 @@ export class ActorExtractLinksSolidTypeIndex extends ActorExtractLinks {
       },
     });
 
+    if (this.inference) {
+      await this.linkPredicateDomain(predicateSubjects, typeSubjects);
+    }
+
     // Check if the current pattern has any of the allowed subjects,
     // and consider the type index entry's links in that case.
     const links: ILink[] = [];
+
```

Comment:
No empty line needed here. (FYI, for these types of issues, it's always a good idea to check your git diff right before or after committing)

Reply:
The empty is still here :/